source/upgrade.txt

```diff
@@ -117,9 +117,9 @@ This driver version introduces the following breaking changes:
   converts automatically to ``org.mongodb.scala.SingleObservable[Void]``. The
   API also exposes ``org.mongodb.scala.Observable[Unit]`` instead of
   ``org.mongodb.scala.Observable[Void]``.
-
-  .. After the 5.0 Scala API docs are released, this line will be uncommented.
-    For more information, see the `Observable trait documentation <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__.
+  
+  For more information, see the `Observable trait documentation
+  <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__.
```

Comment:
S: Not necessary, but I think it would be helpful to further clarify where this links to

```suggestion
  For more information, see the `Observable
  <https://mongodb.github.io/mongo-java-driver/5.0/apidocs/mongo-scala-driver/org/mongodb/scala/Observable.html>`__ trait in the Scala API documentation.
```